docker-compose.yml

```diff
@@ -64,21 +64,21 @@ services:
       - LOGIN_RADIUS_PRIVATE_KEY_PASS
       - LOGIN_RADIUS_KEYSTORE_PASS
       - LOGIN_RADIUS_KEYSTORE_NAME
-      - CIVIFORM_APPLICANT_IDP
-      - CIVIFORM_ADMIN_IDP
+      - CIVIFORM_APPLICANT_IDP=generic-oidc
```

Comment:
Can you expand the PR description with some comments about what this unlocks? If possible, mind adding a quick "Tested" section that describes how this works in `bin/run-dev`?